src/schema.ts

```diff
@@ -273,44 +277,17 @@ export interface FlywheelConnectedEvent {
 }
 
 /**
- * Represents every channel that can be played on the vibraphone.
- *
- * We decided to start at one since the [[BassString]] type also
- * starts at one. We also decided to only represent playable
- * channels and not actual channels (two per note) because that is
- * an implementation detail of the physical MMX.
+ * All possible events relating to the machine itself.
  */
-export type VibraphoneChannel = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11;
+export type MachineEvent = MachineMuteEvent | MachineTempoEvent | FlywheelConnectedEvent;
 
-/**
- * Represents the state pertaining to the vibraphone.
- */
-export interface VibraphoneState {
-	/** Whether or not the vibrato is engaged */
-	vibratoEnabled: boolean;
-	/** The vibrato speed from 0 to 1 */
-	vibratoSpeed: number;
-	/**
-	 * Which notes represent which channels.
-	 * These cannot be changed via events and thus stay
-	 * the same throughout the course of an entire [[Performance]].
-	 * They can however, be changed while editing a [[Program]].
-	 */
-	notes: { [VC in VibraphoneChannel]: Note };
-}
-
-/**
- * Any event pertaining to the vibraphone.
- */
-export type VibraphoneEvent =
-	| VibraphoneVibratoEnabledEvent
-	| VibraphoneVibratoSpeedEvent;
 /** An event corresponding to a change in the [[VibraphoneState.vibratoEnabled]] property. */
 export interface VibraphoneVibratoEnabledEvent {
```

Comment:
I don't know if "Enabled" is a good term for changing from enabled to disabled. Perhaps "StateChange"?

Reply:
I don't think StateChange is better, all events represent a state change, the name of the event represents which property of the state is being changed, it makes for consistent event naming.

Reply:
Hmm.. you're right. "StateChange" isn't better. How about "Engaged"?